### PR TITLE
MIOpen Documentation for ROCm 6.1

### PR DESCRIPTION
**Pull request by Rmalavally**
Updated and restructured MIOpen documentation for ROCm 6.1.

Pending: Doxygen links for API content

### CODE REVIEW

File: docs/install/install.rst
Comment (randyh62) [inline docs/install/install.rst:1]: It looks like the header tags are not correct in this file.

Comment (Rmalavally) [reply docs/install/install.rst:1]: Yes, Randy. Working on it.

File: docs/index.rst
Comment (randyh62) [inline docs/index.rst:1]: It looks like the index needs the card layout: Install, Conceptual, Reference, Tutorials

File: docs/install/install.rst
Comment (Rmalavally) [reply docs/install/install.rst:1]: @bghimireamd Could you please confirm?

File: docs/example.rst
Comment (randyh62) [inline docs/example.rst:1]: Probably can remove this empty file?

File: docs/index.rst
Comment (Rmalavally) [reply docs/index.rst:1]: Yes. Adding index files.

File: docs/conceptual/Getting_Started_FusionAPI.md
Comment (randyh62) [inline docs/conceptual/Getting_Started_FusionAPI.md:1]: Is this the same file as the file named the same in the Tutorials folder?

**Review by randyh62 — COMMENTED** [review]
It looks like the files in the Conceptual folder are the same, or at least named the same as the files in the Tutorials folder? I left some other comments, but think there may be some work to do here.

**Comment by Rmalavally** [comment]
> It looks like the files in the Conceptual folder are the same, or at least named the same as the files in the Tutorials folder? I left some other comments, but think there may be some work to do here.

Yes. Need to clean up.

**Pull request closed without merging**